Binary files wildguppy.orig/about.pyo and wildguppy/about.pyo differ
diff -rupN wildguppy.orig/panel_app.py wildguppy/panel_app.py
--- wildguppy.orig/panel_app.py	2014-02-05 14:45:04.814401110 +0100
+++ wildguppy/panel_app.py	2014-02-05 14:49:38.105890041 +0100
@@ -9,7 +9,7 @@ import gobject
 import json
 
 gobject.threads_init()
-file_path = "/opt/wildguppy/icon/fish.png"
+file_path = "/usr/share/pixmaps/wildguppy-applet.png"
 config_path = os.getenv("HOME")+"/.config/wildguppy/config.json"
 script_dir, script_name = os.path.split(os.path.realpath(__file__))
 
Binary files wildguppy.orig/panel_app.pyo and wildguppy/panel_app.pyo differ
Binary files wildguppy.orig/settings.pyo and wildguppy/settings.pyo differ
diff -rupN wildguppy.orig/wildguppy.desktop wildguppy/wildguppy.desktop
--- wildguppy.orig/wildguppy.desktop	2014-02-05 14:45:04.814401110 +0100
+++ wildguppy/wildguppy.desktop	2014-02-05 15:04:32.454998398 +0100
@@ -2,8 +2,8 @@
 Version=1.0
 Type=Application
 Terminal=false
-Exec=wildguppy-gtk
+Exec=wildguppy-applet
 Name=wildguppy
 Comment=Adjusts a laptop's brightness automatically, by using camera samples taken at a user definable interval.
-Icon=/opt/wildguppy/icon/fish_large.png
+Icon=wildguppy
 Categories=Utility;GTK;GNOME;
diff -rupN wildguppy.orig/wildguppy.py wildguppy/wildguppy.py
--- wildguppy.orig/wildguppy.py	2014-02-05 14:45:04.814401110 +0100
+++ wildguppy/wildguppy.py	2014-02-05 14:45:41.903145570 +0100
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
-import Image
-import ImageStat
+from PIL import Image
+from PIL import ImageStat
 import math
 import os
 import time
Binary files wildguppy.orig/wildguppy.pyo and wildguppy/wildguppy.pyo differ
